src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/MuTect2.java

```diff
@@ -1041,5 +1041,8 @@ public ActivityProfileState isActive(final AlignmentContext context, final Refer
             return new ActivityProfileState( ref.getInterval(), prob, ActivityProfileState.Type.NONE, null);
         }
     }
+
+    public static boolean hasBadMate(final GATKRead rec) {
```

Comment:
To be fair, that could also be the correct mapping for a pair of read supporting a structural variant, which are very common (compared with germline) in some tumor types. MuTect2 can't call SVs, but one of the mates being wrong isn't the only possibility.